docs/reference/deploy/config/master-config-reference.rst

```diff
@@ -1595,3 +1595,11 @@ requests and responses.
 =================
 
 The path to the IdP's certificate, used to validate assertions.
+
```

Comment:
I wasn't able to discern any ordering of the items in this file, so I just appended this.